lib/mutations/errors.rb

```diff
@@ -47,11 +47,21 @@ class DefaultErrorMessageCreator
     #  :index -- index of error if it's in an array
     def message(key, error_symbol, options = {})
       if options[:index]
-        "#{(key || 'array').to_s.titleize}[#{options[:index]}] #{MESSAGES[error_symbol]}"
+        "#{titleize(key || 'array')}[#{options[:index]}] #{MESSAGES[error_symbol]}"
       else
-        "#{key.to_s.titleize} #{MESSAGES[error_symbol]}"
+        "#{titleize(key)} #{MESSAGES[error_symbol]}"
       end
     end
+
+    def titleize(key)
+      return key.to_s.titleize({keep_id_suffix: true}) if titleize_options_supported?
```

Comment:
If we have to have our own implementation I'd prefer to always use it, even when `keep_id_suffix` is available.

Reply:
👍